external/vcm/vcm/cdl/generate.py

```diff
@@ -0,0 +1,113 @@
+from typing import Mapping
+import lark
+import numpy as np
+import xarray
+
+from .parser import grammar
+
```

Comment:
Can you add back the info comment that was in the removed cdl.py pointing to the UCAR site for people to reference how to write CDL strings?

Reply:
I added these docs to the cdl_to_dataset docstring.